README.md

```diff
@@ -1,8 +1,35 @@
 # Gardener Setup Scripts
 
-This is the installation manual for a simple Gardener setup. It is part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
-
-
+This repo serves as the installation manual for a simple Gardener setup. Basically, it explains how you can create a simple cluster with project [Kubify](https://github.com/gardener/kubify) and walks you through the steps required to bootstrap a Gardener cluster in an automated manner (if you just want to run a script) or in a step-by-step manner for those who seek more understanding of whats going on under the hood. 
```

Comment:
No, this README is the manual, the repo contains the installer, and the landscape-setup-template serves as a configuration template for this installer.

typo: s/whats/what's/